scripts/e2e/setup-cluster.sh

```diff
@@ -41,9 +41,9 @@ store_cluster_secret() {
   cat <<EOF |  yq -P - > /tmp/temp_k8sconfig.yaml
 {
     "k8s_config": {
-        "name": $(k3d kubeconfig get "$CLUSTER_NAME" | yq .clusters[0].name -o json -),
+        "name": $(k3d kubeconfig get "$CLUSTER_NAME" | yq '.clusters[0].name' -o json -),
```

Comment:
Added these because the expressions (e.g. `.clusters[0].name`) occasionally throw and error if they are not expressed as an explicit string (depends on the version of `yq` used).